### DISCUSSION TITLE
Issue with preprocessed data + TBSS

### DISCUSSION THREAD

**Discussion by CristinaRoman1** [muscbridge/PyDKE#285] [Technical Support] [answered]
Hello PyDesigner Community- 

I used the PyDesigner default pipeline to preprocess multi-shell (2 b-values, 30 directions each) human diffusion data in preparation for TBSS. The outputted FA, MK, RK, and AK maps look fine upon visual inspection, but when I run the data through TBSS steps 1-3, the registered FA maps and skeletons come out quite distorted. In retracing my steps, I believe it has something to do with the PyDesigner created FA/MK/etc. maps I am feeding into the pipeline. I noticed that BET is not run as pat of the pipeline (while acknowledging a brain mask is in fact created)- should I be running BET on my data independently? Is there something I should be doing with the data after PyDesigner but before TBSS? 

Thanks in advance for your time, 
Cristina

**Comment by TheJaeger** [comment]
Hi Cristina,

To help you better, could you post the the version and command you used to process with PyDesigner?

**Comment by CristinaRoman1** [comment]
Sure. We downloaded PyDesigner a few weeks ago, so I presume we are using
the current version. We are running the following command: nohup pydesigner
-s [diffusion data] > pydesigner_output.txt &

On Sun, Jan 23, 2022 at 2:58 AM Siddhartha Dhiman ***@***.***>
wrote:

> Hi Cristina,
>
> To help you better, could you post the the version and command you used to
> process with PyDesigner?
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/m-ama/PyDesigner/discussions/285#discussioncomment-2026492>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AQI3HZUK4ISNSDN66XN7V6DUXOYK5ANCNFSM5MS72H4A>
> .
> Triage notifications on the go with GitHub Mobile for iOS
> <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675>
> or Android
> <https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.
>
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>


-- 
*Cristina A. F. Rom**án, Ph.D. *

National MS Society Fellow

Neuropsychology and Neuroscience Lab

Kessler Foundation

Affiliation: Assistant Professor (Research), Department of Physical
Medicine & Rehabilitation and of Neurology, Rutgers New Jersey Medical
School

1199 Pleasant Valley Way

West Orange, NJ 07052
P. 973.965.6637
E: ***@***.***
Twitter: @CristinaAFRoman
pronouns: she/her(s)/ella

“*We cannot seek achievement for ourselves and forget about progress and
prosperity for our*

*community…Our ambitions must be broad enough to include the aspirations
and needs of others,*

*for their sakes and for our own*.”

-César Chávez

________________________________

*Confidentiality Notice*

The information in this email is intended *only for the individual or
entity to which it is addressed* and may contain information that is
privileged or confidential. If you are not the intended recipient, or the
employee or agent responsible for delivering the message to the intended
recipient, you are hereby notified that any dissemination, distribution or
copying of this communication is strictly prohibited, and you are requested
to please notify us immediately by telephone or email, and destroy the
original message.

**Comment by CristinaRoman1** [comment, marked as answer]
I think I figured out the problem. The DTI and DKI maps have NaN values
(looks like within the FoV) that TBSS has trouble dealing with, thereby
impacting the registration and transformation steps. I ran a test with 10
participants where I removed all NaN values from the FA/MK/etc. maps and
TBSS steps 1-3 look good.

Thanks again for your time.
Cristina

On Sun, Jan 23, 2022 at 1:26 PM Cristina A. F. Román <
***@***.***> wrote:

> Sure. We downloaded PyDesigner a few weeks ago, so I presume we are using
> the current version. We are running the following command: nohup
> pydesigner -s [diffusion data] > pydesigner_output.txt &
>
> On Sun, Jan 23, 2022 at 2:58 AM Siddhartha Dhiman <
> ***@***.***> wrote:
>
>> Hi Cristina,
>>
>> To help you better, could you post the the version and command you used
>> to process with PyDesigner?
>>
>> —
>> Reply to this email directly, view it on GitHub
>> <https://github.com/m-ama/PyDesigner/discussions/285#discussioncomment-2026492>,
>> or unsubscribe
>> <https://github.com/notifications/unsubscribe-auth/AQI3HZUK4ISNSDN66XN7V6DUXOYK5ANCNFSM5MS72H4A>
>> .
>> Triage notifications on the go with GitHub Mobile for iOS
>> <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675>
>> or Android
>> <https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.
>>
>> You are receiving this because you authored the thread.Message ID:
>> ***@***.***>
>>
>
>
> --
> *Cristina A. F. Rom**án, Ph.D. *
>
> National MS Society Fellow
>
> Neuropsychology and Neuroscience Lab
>
> Kessler Foundation
>
> Affiliation: Assistant Professor (Research), Department of Physical
> Medicine & Rehabilitation and of Neurology, Rutgers New Jersey Medical
> School
>
> 1199 Pleasant Valley Way
>
> West Orange, NJ 07052
> P. 973.965.6637
> E: ***@***.***
> Twitter: @CristinaAFRoman
> pronouns: she/her(s)/ella
>
> “*We cannot seek achievement for ourselves and forget about progress and
> prosperity for our*
>
> *community…Our ambitions must be broad enough to include the aspirations
> and needs of others,*
>
> *for their sakes and for our own*.”
>
> -César Chávez
>
> ________________________________
>
> *Confidentiality Notice*
>
> The information in this email is intended *only for the individual or
> entity to which it is addressed* and may contain information that is
> privileged or confidential. If you are not the intended recipient, or the
> employee or agent responsible for delivering the message to the intended
> recipient, you are hereby notified that any dissemination, distribution or
> copying of this communication is strictly prohibited, and you are requested
> to please notify us immediately by telephone or email, and destroy the
> original message.
>


-- 
*Cristina A. F. Rom**án, Ph.D. *

National MS Society Fellow

Neuropsychology and Neuroscience Lab

Kessler Foundation

Affiliation: Assistant Professor (Research), Department of Physical
Medicine & Rehabilitation and of Neurology, Rutgers New Jersey Medical
School

1199 Pleasant Valley Way

West Orange, NJ 07052
P. 973.965.6637
E: ***@***.***
Twitter: @CristinaAFRoman
pronouns: she/her(s)/ella

“*We cannot seek achievement for ourselves and forget about progress and
prosperity for our*

*community…Our ambitions must be broad enough to include the aspirations
and needs of others,*

*for their sakes and for our own*.”

-César Chávez

________________________________

*Confidentiality Notice*

The information in this email is intended *only for the individual or
entity to which it is addressed* and may contain information that is
privileged or confidential. If you are not the intended recipient, or the
employee or agent responsible for delivering the message to the intended
recipient, you are hereby notified that any dissemination, distribution or
copying of this communication is strictly prohibited, and you are requested
to please notify us immediately by telephone or email, and destroy the
original message.

**Reply by TheJaeger** [reply]
Most FSL tools do not enjoy the presence of NaNs. We always run `fslmaths` to remove NaNs as the first step if we were to use FSL in any of our pipelines. I'm happy you figured it out and got your pipeline to work as intended.

P.S. It would be awesome if you could post some metrics images in "Show and Tell" section of Discussions. I suppose this is 7T human data so it could be revealing.

**Comment by CristinaRoman1** [comment]
Gotcha, thanks!

I have not gotten to the 7T diffusion data yet, but if/once I do, I can
post what we come up with. :)

On Mon, Jan 24, 2022 at 9:28 AM Siddhartha Dhiman ***@***.***>
wrote:

> Most FSL tools do not enjoy the presence of NaNs. We always run fslmaths
> to remove NaNs as the first step if we were to use FSL in any of our
> pipelines. I'm happy you figured it out and got your pipeline to work as
> intended.
>
> P.S. It would be awesome if you could post some metrics images in "Show
> and Tell" section of Discussions. I suppose this is 7T human data so it
> could be revealing.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/m-ama/PyDesigner/discussions/285#discussioncomment-2034438>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AQI3HZVJX5UEEK32GHLTUVLUXVOZFANCNFSM5MS72H4A>
> .
> Triage notifications on the go with GitHub Mobile for iOS
> <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675>
> or Android
> <https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.
>
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>


-- 
*Cristina A. F. Rom**án, Ph.D. *

National MS Society Fellow

Neuropsychology and Neuroscience Lab

Kessler Foundation

Affiliation: Assistant Professor (Research), Department of Physical
Medicine & Rehabilitation and of Neurology, Rutgers New Jersey Medical
School

1199 Pleasant Valley Way

West Orange, NJ 07052
P. 973.965.6637
E: ***@***.***
Twitter: @CristinaAFRoman
pronouns: she/her(s)/ella

“*We cannot seek achievement for ourselves and forget about progress and
prosperity for our*

*community…Our ambitions must be broad enough to include the aspirations
and needs of others,*

*for their sakes and for our own*.”

-César Chávez

________________________________

*Confidentiality Notice*

The information in this email is intended *only for the individual or
entity to which it is addressed* and may contain information that is
privileged or confidential. If you are not the intended recipient, or the
employee or agent responsible for delivering the message to the intended
recipient, you are hereby notified that any dissemination, distribution or
copying of this communication is strictly prohibited, and you are requested
to please notify us immediately by telephone or email, and destroy the
original message.

**Comment by DaveGeneral** [comment]
这是来自QQ邮箱的假期自动回复邮件。已收到，-_-
